packages/amp/src/index.tsx

```diff
@@ -118,6 +126,11 @@ const amp: AMP = {
       },
     },
   },
-};
+  libraries: {
+    html2react: {
+      processors,
+    },
+  },
+} as AMP;
```

Comment:
I don't think this is needed, right?

```suggestion
};
```

Reply:
nope, not needed. 👍